app/models/sponsor_group.rb

```diff
@@ -0,0 +1,6 @@
+class SponsorGroup < ActiveRecord::Base
+  has_one :application
+
+  validates :citizenship_status, inclusion: { in: ["Canadian Citizen", "Permanent Resident", "Other"] }
+  validates_presence_of :name, :email, :phone
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.